src/applications/personalization/profile-2/components/connected-apps/components/AppDeletedAlert.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export class AppDeletedAlert extends React.Component {
```

Comment:
I certainly don't mind this being a class but it would be pretty easy to make this a functional component with `useEffect` and `useRef` if you wanted to spend even more time on this PR!

Reply:
Done! You're right, it's a good idea to switch to using hooks and get more familiar 👍